.github/workflows/parsl+flux.yaml

```diff
@@ -0,0 +1,47 @@
+name: Test Flux Scheduler
+on:
+  pull_request: []
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    permissions:
```

Comment:
This is now 4 years out of date.  Are we interested in using a more recent distro for the test and CI framework?

Reply:
The flux container image is tagged for `jammy` which is the subsequent ubuntu LTS release (22.04).  The most recent ubuntu LTS is 24.04 which was released just over a month ago, and it looks like there isn't a flux container for that (on https://hub.docker.com/r/fluxrm/flux-sched/tags).

So I'll upgrade this to 22.04 and see what happens.